demo/movielens/online/src/main/java/com/dmetasoul/metaspore/demo/movielens/diversify/diversifier/DiverseProvider.java

```diff
@@ -31,18 +31,10 @@ public DiverseProvider(List<Diversifier> diversifiers) {
         diversifiers.forEach(x -> diversifierMap.put(x.getClass().getSimpleName().toLowerCase(), x));
     }
 
-    public Diversifier getDiversifier(String name) {
-        return diversifierMap.get(name.toLowerCase());
-    }
-
-    public Collection<Diversifier> getDiversifiers(Collection<String> names) {
-        if (names == null) {
-            return Collections.emptyList();
+    public Diversifier getDiversifiers(String name) {
+        if (name == null) {
+            return diversifierMap.get("simplediersifier");
         }
-        ArrayList<Diversifier> diversifiers = new ArrayList<>();
-        for (String name : names) {
-            diversifiers.add(getDiversifier(name));
-        }
-        return diversifiers;
+        return diversifierMap.get(name.toLowerCase());
```

Comment:
why delete 
`public Collection<Diversifier> getDiversifiers(Collection<String> names)`
this function?